data/tlg0062/tlg001/tlg0062.tlg001.perseus-eng1.xml

```diff
@@ -0,0 +1,683 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">Phalaris</title>
+<author>Lucian of Samosata</author>
+<editor>Austin Morris Harmon, 1878-1950</editor>
+<funder>National Endowment for the Humanities</funder>
+<principal>Gregory Crane</principal>
+<respStmt>
+<persName>Gregory Crane</persName>
+<resp>OCRd, tagged and added initial corrections to  the text</resp>
+</respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>Perseus Digital Library</authority>
+<idno type="filename">tlg0062.tlg001.1st1K-eng1.xml</idno>
+<availability>
+<licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+</availability>
+<date>2023</date>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>United States</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+<biblStruct>
+<monogr>
+<title>Lucian</title>
+<editor>
+<persName>
+<name>Austin Morris Harmon</name>
+</persName>
+</editor>
+<author ref="urn:cts:greekLit:tlg0062">Lucian of Samosata</author>
+<imprint>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>Cambridge, MA</pubPlace>
+<date>1913</date>
+</imprint>
+<biblScope unit="volume">1</biblScope>
+</monogr>
+<ref target="https://hdl.handle.net/2027/uc1.b000604153">The Hathi Trust</ref>
+</biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+<p>Text encoded in accordance with the latest EpiDoc standards</p>
+<refsDecl n="CTS">
+    <cRefPattern matchPattern="(\w+).(\w+)"
+     replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])">
+     <p>This pointer pattern extracts book and section</p>
+    </cRefPattern>
+    <cRefPattern matchPattern="(\w+)"
+     replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])">
+     <p>This pointer pattern extracts book</p>
+    </cRefPattern>
+   </refsDecl>
+  </encodingDesc>
+<profileDesc>
+<langUsage>
+<language ident="grc">Greek</language>
+<language ident="lat">Latin</language>
+</langUsage>
+</profileDesc>
+<revisionDesc>
+<change/>
+</revisionDesc>
+</teiHeader>
+<text>
+<body>
+<div type="translation" n="urn:cts:greekLit:tlg0062.tlg001.perseus-eng1" xml:lang="eng">
+<head>Phalaris<note>This piece and its fellow should not be taken as a serious attempt to whitewash Phalaris and to excuse Delphi for accepting a tainted gift. They are good specimens of the stock of a rhetorician, and something more. To put yourself in another man’s shoes and say what he would have said was a regular exercise of the schools, but to laugh in your sleeve as you said it was not the way of the ordinary rhetorician.</note></head>
+<div type="textpart" xml:base="urn:cts:greekLit:tlg0062.tlg001.perseus-eng1" subtype="book" n="1">
+<head>I</head>
+<div type="textpart" xml:base="urn:cts:greekLit:tlg0062.tlg001.perseus-eng1" subtype="section" n="1">
```

Comment:
This xml:base is incorrect. You are one step down the tree and missing the book .
`xml:base="urn:cts:greekLit:tlg0062.tlg001.perseus-eng1"`
should be
`xml:base="urn:cts:greekLit:tlg0062.tlg001.perseus-eng1:1"` where the `:1` indicates this section is in book 1.

Without drilling down, the xml:base is superfluous.